frontend/src/OnBoardingForm/OnBoardingForm.jsx

```diff
@@ -22,7 +22,7 @@ function OnBoardingForm({ userDetails = {}, token = '', organizationToken = '',
     companyName: '',
     role: '',
     companySize: '',
-    phoneNumber: '',
+    phoneNumber: '+1',
```

Comment:
I tried it and this is not working. You should set a default county as "US" in phone input.